data/kebijakan-privasi.json

```diff
@@ -15,19 +15,19 @@
       },
       {
         "title":"Cookies",
-        "description":"<p>Situs kami tidak menggunakan cookies secara eksplisit. Namun, Aplikasi mungkin menggunakan kode dan <em>library</em> pihak ketiga yang menggunakan <em>cookies</em> untuk mengumpulkan informasi dan memperbaiki layanan mereka. Pengguna dapat menginstruksikan browser mereka, dengan mengubah pilihan, untuk berhenti menerima <em>cookies</em> atau untuk meminta Pengguna sebelum menerima <em>cookies</em> dari situs-situs yang dikunjungi. Jika Pengguna tidak menerima <em>cookies</em>, maka Pengguna mungkin tidak dapat menggunakan semua bagian dari Situs dan Aplikasi atau semua fungsi dari layanan kami.</p>"
+        "description":"<p>Situs kami tidak menggunakan <em>cookies</em> secara eksplisit. Namun, Aplikasi mungkin menggunakan kode dan <em>library</em> pihak ketiga yang menggunakan <em>cookies</em> untuk mengumpulkan informasi dan memperbaiki layanan mereka. Pengguna dapat menginstruksikan browser mereka, dengan mengubah pilihan, untuk berhenti menerima <em>cookies</em> atau untuk meminta Pengguna sebelum menerima <em>cookies</em> dari situs-situs yang dikunjungi. Jika Pengguna tidak menerima <em>cookies</em>, maka Pengguna mungkin tidak dapat menggunakan semua bagian dari Situs dan Aplikasi atau semua fungsi dari layanan kami.</p>"
```

Comment:
Hello, @marsellavaleria19 can you explain why we need to store an HTML string inside a JSON file? do you intend to render this HTML string using v-html? 🤔

Reply:
Hello @Ibwedagama, yes I will render using v-html, because any details have a different layout, like this 
![image](https://user-images.githubusercontent.com/44119106/196614397-3d7e451e-be23-4384-8721-0ef5f81a9094.png)
![image](https://user-images.githubusercontent.com/44119106/196614637-51f2da2d-96fe-4195-bf38-432fe0f420aa.png)